web-console/future-openshift/diagnostics/diagnostics.md

```diff
@@ -0,0 +1,37 @@
+# Diagnostic Report
+
+Users can download a diagnostic report of their system and send it to Red Hat from the console.
+
+
+## Help menu
+![dropdown](img/dropdown.png)
+* The "Diagnostic Report" option is available from the help menu.
+
+## Modal
+![modal](img/modal.png)
+* A modal will open with a description of how the diagnostic report will be used.
+* Users can indicate what they would like to include in the diagnostic report
+  * If users choose to include node health, a multiselect dropdown will allow them to select which ones.
+* Users can indicate whether or not they have an open support case already.
+  * If users have a case number they can enter it into the "Case number" field.
```

Comment:
Will this attach it to the case AND download? I'm assuming yes since the overall action is "Download". So maybe we should say something in this support case options like "Download and attach to existing support case" and "Download and open a new support case" or something along those lines :)